### ISSUE TITLE
interactive mode?

### ISSUE THREAD

**Issue by nitsanavni** [stateful/runme#38]
cool project!
WDY about an interactive cli?

**Comment by adambabik** [comment]
@nitsanavni thank you! That's one of the options we consider. More broadly, we are looking into long-lived interactive sessions similar to a concept of Kernel from Jupyter Notebooks.

**Comment by lalyos** [comment]
You can achieve it with the current runme with theh elp of [fzf](https://github.com/junegunn/fzf) [jq](https://stedolan.github.io/jq/) and xargs:
```
runme json \
 | jq '.document[]|select(.name)|.name' -r \
 | fzf --preview='runme print {}' --layout=reverse --height=80% -e \
 | xargs runme run
```

I made a small demo: [![asciicast](https://asciinema.org/a/mFrLGD6crII1pAaFzhkGJ75US.svg)](https://asciinema.org/a/mFrLGD6crII1pAaFzhkGJ75US)

**Comment by adambabik** [comment]
We just released version v0.5.0 which has an interactive mode! 🎉  

Please update runme (confirm with `runme --version` that it is v0.5.0 indeed) and try running `runme` or `runme tui`. Hope you will like it!
